src/enums/authorizedActions.ts

```diff
@@ -0,0 +1,26 @@
+/** List of actions authorized via roles. See Authorizations module. */
+export enum AuthorizedActions {
+  FOLIO_NUMBER,
+  SBC_BREADCRUMBS,
+  STAFF_BREADCRUMBS,
+  STAFF_COMMENTS,
+  BLANK_COMPLETING_PARTY,
+  BLANK_CERTIFY_STATE,
+  EDITABLE_EMAIL_ADDRESS,
+  EDITABLE_COMPLETING_PARTY,
+  EDITABLE_CERTIFY_NAME,
+  THIRD_PARTY_CERTIFY_STMT,
+  NO_COMPLETING_PARTY_MESSAGE_BOX,
+  NO_MIN_START_DATE,
+  NO_CONTACT_INFO,
+  COURT_ORDER_POA,
+  FIRM_COMPLETING_PARTY,
+  STAFF_PAYMENT,
+  FILE_AND_PAY,
+  AML_OVERRIDES,
+  ADD_EDIT_BUSINESS,
+  ADD_UNREGISTERED_BUSINESS,
+  ADD_REGISTERED_BUSINESS,
+  CONTINUATION_IN_FILING,
+  RESTORATION_FILING,
+}
```

Comment:
I'll work with Mihai and try to combine some of these.

However, this PR is trying to be a refactor only (like for like).

Reply:
Thoughts on whether I should rename FIRM* to REG* ? (Same as components that use them.)

Reply:
Looks great Severin

Reply:
No change here so far. Resolving. I'll open a new conversation if neeed.